test/thermodynamics/work_of_ideal_gas_in_isothermal_process_test.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    assert_equal
+)
+
+from symplyphysics.laws.thermodynamics import work_of_ideal_gas_in_isothermal_process as work_of_ideal_gas
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    mass = Quantity(0.5 * units.kilogram)
+    molar_mass = Quantity(0.028 * units.kilogram / units.mole)
+    start_volume = Quantity(32 * units.meter**3)
+    final_volume = Quantity(48 * units.meter**3)
+    temperature = Quantity(273 * units.kelvin)
```

Comment:
Do you have any link to verify the numbers?

Reply:
No, I took data from different tasks

Reply:
I suggest to look for some external source of data because there are some conventions about sign of the work, depending on who makes the work.